content/docs/2.9/scalers/azure-event-hub.md

```diff
@@ -68,7 +74,9 @@ spec:
     provider: Azure | azure-workload
 ```
 
-When you do so, the Event Hub scaler will depend on the existence of two configurations you have to provide: `eventHubNamespace` and `eventHubName`.
+When you do so, the Event Hub scaler will depend on the existence of two configurations you have to provide: `eventHubNamespace` and `eventHubName`. You can also configure `storageAccountName` if you wish to use Azure AD Pod Identity to authenticate to Azure Blob Storage instead of a connection string. 
```

Comment:
Sorry for the trouble here, a minor change.

```suggestion
When you do so, the Event Hub scaler will depend on the existence of two configurations you have to provide: `eventHubNamespace` and `eventHubName`. You can also configure `storageAccountName` if you wish to use Azure AD Pod / Workload Identity to authenticate to Azure Blob Storage instead of a connection string. 
```